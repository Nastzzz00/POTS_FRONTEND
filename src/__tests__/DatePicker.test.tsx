/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
// import ShallowRenderer from "react-test-renderer/shallow";

// // Note: test renderer must be required after react-native.
// import renderer from "react-test-renderer";

import HeaderTitle from "../components/HeaderTitle";
import DatePickerComponent from "../components/DatePicker";

const props = {
  title: "Date Picker"
};

describe("Date Picker", () => {
  test("renders correctly", () => {
    expect(shallow(<DatePickerComponent {...props} />)).toMatchSnapshot();
  });
});
