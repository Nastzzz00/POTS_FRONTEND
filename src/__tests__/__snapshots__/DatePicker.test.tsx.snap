// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date Picker renders correctly 1`] = `ShallowWrapper {}`;
